--- origsrc/hunspell-1.2.6/configure.ac	2008-07-15 03:43:12.000000000 -0500
+++ src/hunspell-1.2.6/configure.ac	2008-07-16 11:19:13.484375000 -0500
@@ -41,6 +41,7 @@
 
 dnl internationalization macros
 AM_GNU_GETTEXT
+AM_GNU_GETTEXT_VERSION([0.15])
 
 AC_ARG_WITH(warnings,[  --with-warnings         compile with warning messages], [
      AC_DEFINE(HUNSPELL_WARNING_ON,1,"Define if you need warning messages")
--- origsrc/hunspell-1.2.6/src/hunspell/Makefile.am	2008-06-17 03:57:54.000000000 -0500
+++ src/hunspell-1.2.6/src/hunspell/Makefile.am	2008-07-16 11:19:13.484375000 -0500
@@ -1,6 +1,7 @@
 #libhunspell_la_LDFLAGS = -version-info x:x:x
 
 lib_LTLIBRARIES = libhunspell-1.2.la
+libhunspell_1_2_la_LDFLAGS = -no-undefined
 libhunspell_1_2_includedir = $(includedir)/hunspell
 libhunspell_1_2_la_SOURCES=affentry.cxx affixmgr.cxx csutil.cxx \
 		     dictmgr.cxx hashmgr.cxx hunspell.cxx utf_info.cxx \
--- origsrc/hunspell-1.2.6/src/parsers/Makefile.am	2007-11-16 02:50:27.000000000 -0600
+++ src/hunspell-1.2.6/src/parsers/Makefile.am	2008-07-16 11:19:13.500000000 -0500
@@ -1,4 +1,4 @@
-lib_LIBRARIES=libparsers.a
+noinst_LIBRARIES=libparsers.a
 libparsers_a_SOURCES=firstparser.cxx htmlparser.cxx \
 		      latexparser.cxx  manparser.cxx \
 		      textparser.cxx 
--- origsrc/hunspell-1.2.6/src/tools/Makefile.am	2008-04-04 08:39:21.000000000 -0500
+++ src/hunspell-1.2.6/src/tools/Makefile.am	2008-07-16 11:19:13.515625000 -0500
@@ -1,4 +1,4 @@
-bin_PROGRAMS=analyze chmorph example hunspell munch unmunch hzip hunzip
+bin_PROGRAMS=hunspell-analyze hunspell-chmorph hunspell-example hunspell hunspell-munch hunspell-unmunch hzip hunzip
 
 INCLUDES=-I${top_srcdir}/src/hunspell -I${top_srcdir}/src/parsers
 
@@ -6,23 +6,21 @@
 hunzip_SOURCES=hunzip.cxx
 hunzip_LDADD = ../hunspell/libhunspell-1.2.la
 
-munch_SOURCES=munch.c
-unmunch_SOURCES=unmunch.c
-include_HEADERS=munch.h unmunch.h
+hunspell_munch_SOURCES=munch.c
+hunspell_unmunch_SOURCES=unmunch.c
+pkginclude_HEADERS=munch.h unmunch.h
 
-example_SOURCES=example.cxx
-example_LDADD = ../hunspell/libhunspell-1.2.la
+hunspell_example_SOURCES=example.cxx
+hunspell_example_LDADD = ../hunspell/libhunspell-1.2.la
 
 hunspell_SOURCES=hunspell.cxx
-hunspell_LDADD = @LIBINTL@ ../hunspell/libhunspell-1.2.la \
-	../parsers/libparsers.a @CURSESLIB@ @READLINELIB@
+hunspell_LDADD = ../parsers/libparsers.a ../hunspell/libhunspell-1.2.la \
+	@LTLIBINTL@ @LTLIBICONV@ @CURSESLIB@ @READLINELIB@
 
-analyze_SOURCES=analyze.cxx
-analyze_LDADD = ../hunspell/libhunspell-1.2.la
+hunspell_analyze_SOURCES=analyze.cxx
+hunspell_analyze_LDADD = ../hunspell/libhunspell-1.2.la
 
-chmorph_SOURCES=chmorph.cxx
-chmorph_LDADD = ../hunspell/libhunspell-1.2.la ../parsers/libparsers.a
-
-noinst_PROGRAMS=example
+hunspell_chmorph_SOURCES=chmorph.cxx
+hunspell_chmorph_LDADD = ../parsers/libparsers.a ../hunspell/libhunspell-1.2.la
 
 EXTRA_DIST=makealias affixcompress
--- origsrc/hunspell-1.2.6/src/tools/example.cxx	2008-06-17 06:07:24.000000000 -0500
+++ src/hunspell-1.2.6/src/tools/example.cxx	2008-07-16 11:19:13.531250000 -0500
@@ -17,8 +17,8 @@
     /* first parse the command line options */
 
     if (argc < 4) {
-        fprintf(stderr,"example (now it works with more dictionary files):\n"); 
-        fprintf(stderr,"example affix_file dictionary_file(s) file_of_words_to_check\n");
+        fprintf(stderr,"hunspell-example (now it works with more dictionary files):\n"); 
+        fprintf(stderr,"hunspell-example affix_file dictionary_file(s) file_of_words_to_check\n");
         exit(1);
     }
   
--- origsrc/hunspell-1.2.6/src/tools/munch.c	2007-04-06 03:05:28.000000000 -0500
+++ src/hunspell-1.2.6/src/tools/munch.c	2008-07-16 11:19:13.531250000 -0500
@@ -42,14 +42,14 @@
        wf = mystrdup(argv[1]);
   } else {
     fprintf(stderr,"correct syntax is:\n"); 
-    fprintf(stderr,"munch word_list_file affix_file\n");
+    fprintf(stderr,"hunspell-munch word_list_file affix_file\n");
     exit(1);
   }
   if (argv[2]) {
        af = mystrdup(argv[2]);
   } else {
     fprintf(stderr,"correct syntax is:\n"); 
-    fprintf(stderr,"munch word_list_file affix_file\n");
+    fprintf(stderr,"hunspell-munch word_list_file affix_file\n");
     exit(1);
   }
 
--- origsrc/hunspell-1.2.6/src/tools/unmunch.c	2007-04-06 03:05:28.000000000 -0500
+++ src/hunspell-1.2.6/src/tools/unmunch.c	2008-07-16 11:19:13.546875000 -0500
@@ -39,14 +39,14 @@
        wf = mystrdup(argv[1]);
   } else {
     fprintf(stderr,"correct syntax is:\n"); 
-    fprintf(stderr,"unmunch dic_file affix_file\n");
+    fprintf(stderr,"hunspell-unmunch dic_file affix_file\n");
     exit(1);
   }
   if (argv[2]) {
        af = mystrdup(argv[2]);
   } else {
     fprintf(stderr,"correct syntax is:\n"); 
-    fprintf(stderr,"unmunch dic_file affix_file\n");
+    fprintf(stderr,"hunspell-unmunch dic_file affix_file\n");
     exit(1);
   }
 
--- origsrc/hunspell-1.2.6/tests/test.sh	2007-10-27 09:58:42.000000000 -0500
+++ src/hunspell-1.2.6/tests/test.sh	2008-07-16 11:19:13.562500000 -0500
@@ -34,7 +34,7 @@
 shopt -s expand_aliases
 
 alias hunspell='../src/tools/hunspell'
-alias analyze='../src/tools/analyze'
+alias analyze='../src/tools/hunspell-analyze'
 
 if [ "$VALGRIND" != "" ]; then
   rm -f $TEMPDIR/test.pid*
